Store/Store.css

```
@@ -0,0 +1,220 @@
+    format("woff2");
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 85px;
+  background-color: #f9eeed;
+  font-family: "Ansungtangmyun-Bold";
+  font-size: 20px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #ffa4a0;
+  color: ivory;
+  height: 85px;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 1;
+}
+
+header h2 {
+  margin-left: 20px;
+}
+
+#icon {
+  font-size: 30px;
+  margin-top: 25px;
+  margin-right: 20px;
+  position: relative;
+  z-index: 3;
+  color: #edf3c3;
+  cursor: pointer;
+}
+
+.menuBar {
+  display: none;
+  color: #ffa4a0;
+  font-size: 20px;
+  text-align: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 2;
+}
+
+#icon:hover ~ .menuBar,
+.menuBar:hover {
+  width: 250px;
+  height: 100vh;
+  margin: 0;
+  background-color: #edf3c3;
+  display: block;
+}
+
+.menuBar p:hover {
+  color: #acbd98;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  left: 15px;
+  top: 95px;
+  height: 250px;
+  width: 200px;
+  border-radius: 30px;
+  background-color: #bdd1a6;
+}
+
+nav .title {
+  padding-left: 20px;
+  margin-bottom: -10px;
+  color: ivory;
+}
+
+nav .list {
+  list-style-type: none;
+  margin-left: -30px;
+  margin-right: 10px;
+}
+
+li {
+  background-color: #fdf291;
+  color: #666666;
+  border-radius: 20px;
+  padding-left: 10px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  margin-bottom: 10px;
+  cursor: pointer;
+}
+
+li:hover {
+  background-color: #94baf3;
+  opacity: 0.8;
+  color: ivory;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-left: 210px;
+}
+
+article {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  height: 300px;
+  border-radius: 20px;
+  background-color: white;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  box-shadow: 2px 2px 2px 2px rgb(248, 187, 167);
+}
+
+@media (max-width: 872px) {
+  article {
+    width: 400px;
+    margin-right: 0;
+  }
+}
+
+@media (max-width: 700px) {
+  article {
+    width: 330px;
+    margin-right: 0;
+  }
+}
+
+@media (max-width: 610px) {
+  article {
+    width: 270px;
+    margin-right: 0;
+  }
+}
+
+@media (max-width: 550px) {
+  article {
+    width: 225px;
+    margin-right: 0;
+  }
+}
+
+.name {
+  margin-bottom: -10px;
+}
+
+.tag_wrap {
+  position: relative;
+  text-align: center;
+}
+
+.tag {
+  overflow: hidden;
+  white-space: nowrap;
+  word-break: keep-all;
+  width: 9rem;
+  font-size: 15px;
+  color: #666666;
+}
+
+img {
+  height: 150px;
+  width: 150px;
+}
+
+.heart {
+  position: relative;
+  left: 80px;
+  color: rgb(248, 187, 167);
+  cursor: pointer;
+}
+
+@media (max-width: 872px) {
+  .heart {
+    left: 150px;
+  }
+}
```

위에 있는 미디어쿼리랑 합쳐서 요렇게 적는게 더 깔끔할 것 같아요!ㅎㅎ

```suggestion
@media (max-width: 872px) {
  article {
    width: 400px;
    margin-right: 0;
  }
  .heart {
    left: 150px;
  }
}
```

---

안그래도 더 깔끔하게 하고 싶었는데 참고할게요!!:-)